docs/build/examples/meme-marketplace/step-1-blockchain-and-contracts-setup.md

```diff
@@ -0,0 +1,113 @@
+
+```bash
+npm i -g ganache-cli
+ganache-cli
+```
+
+As you can see below, the ganache-cli creates 10 funded accounts (100 ETH in each account address). This tutorial uses one of these addresses and private key pair for transactions.
+
+::: tip
+Mac and Linux users may have to use "sudo" for installing the ganache-cli globally.
+:::
+
+If everything went as it should, your terminal displays messages like those below.
+
+![Ganache CLI initialization output](./images/ganache-cli.png)
+
+6. For use later in this tutorial, copy the "Mnemonic" and save that somewhere you can refer to it later as needed.
+
+In the above screenshot, the "Mnemonic" is:
+
+```bash
+congress elder barrel spice tent advance head manual dragon spice boring bone
+```
+
+::: tip
+NOTE: The "Mnemonic" will be different for you, so use yours. Read this [article](https://kb.myetherwallet.com/en/security-and-privacy/what-is-a-mnemonic-phrase/) to learn more about mnemonics.
+:::
+
+7. Note that the local blockchain exposes an RPC endpoint at `127.0.0.1:8545`. Copy this as well where you can save it, so you can use this endpoint to deploy the contract and interact with the smart contract deployed on the blockchain via this RPC endpoint.
+
+Now, as you have the local blockchain running you need to point your Truffle to the local blockchain RPC URL. In our case, the endpoint is `127.0.0.1:8545` (as shown in the above image). This is already configured on the project in the [truffle-config.js](https://github.com/filecoin-shipyard/meme-nft-token/blob/master/truffle-config.js) file.
+
+8. To deploy the smart contract, open your terminal, and go into the root folder of the project. Run the following command:
+
+```bash
+truffle migrate
+```
+
+::: tip
+To learn more about [compiling](https://www.trufflesuite.com/docs/truffle/getting-started/compiling-contracts) and [deploying](https://www.trufflesuite.com/docs/truffle/getting-started/running-migrations) contracts, refer to Truffle’s documentation.
+:::
+
+If everything went as it should, your terminal returns something similar to the screen below:
+
+![Truffle contract deployment output](./images/contract-deployment.png)
+
+The thing to note here is the "contract address", which is `0x69137609f09bE9eA3C82022D2122642a27a1d7a8` in our case. **This will be different for you.**
+
+9. Copy **and save the contract address somewhere safe**. You will need it in the next sections of the tutorial.
+
+10. To manage the keys and sign transactions, we will use [Metamask](http://metamask.io/). [Download and install Metamask](https://metamask.io/download.html) from their website.
+
+The following video explains how to:
+
+- Download and install Metamask plugin.
+- Importing an existing wallet (using the mnemonic that we saved above).
+- Configuring the Metamask plugin to connect to the local blockchain network (`127.0.0.1:8545`)
+
+@[youtube](https://youtu.be/vbcy55J5xlU)
```

Comment:
I get an error trying to load this video inline, could you check the youtube settings to see if embedding is allowed?

Reply:
Not sure why this is happening. Embedding is already allowed.

![embedding](https://user-images.githubusercontent.com/28847087/89201130-a9fb0100-d5ce-11ea-96a5-b4a6419a22ff.png)

Reply:
BTW, I have also uploaded an [mp4 file for the video](https://github.com/filecoin-project/filecoin-docs/pull/179/files#diff-17e26786ba1ee613f73cb66cf0b0bfce). We can use this if the youtube link doesn't work :)

Reply:
checking again - the issue here could be because of the short url, could you try using `https://www.youtube.com/watch?v=vbcy55J5xlU` instead.

Also please remove the mp4 from this branch as it adds too much weight to the git history.

Reply:
Done. The youtube video is working, and the mp4 file is removed :)